app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= @guest.name %>'s page</h1>
+<H3>Rooms stayed in</h3>
+<% @guest.rooms.each do |room| %>
+  <p>Rooms:  <%= room.suite %> </p>
+  <p>Rate:  <%= room.rate %> </p>
+  <p>Hotel:  <%= room.hotel.name %> </p>
+<% end %>
+
+<h3>Add Room</h3>
+<%=form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |f|%>
+  <%=f.label "ID: " %>
```

Comment:
Don't forget to add a "name" attribute to your labels as well. It should be the same as the field that the label is for.

```ruby
<%= f.label :room_id, "ID: " %>
```